src/pykeen/models/unimodal/triple_re.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+
+"""Implementation of TripleRE."""
+
+from collections import ChainMap
+from typing import Any, ClassVar, Mapping, Optional
+
+from torch.nn import functional
+from torch.nn.init import uniform_
+
+from ..nbase import ERModel
+from ...constants import DEFAULT_EMBEDDING_HPO_EMBEDDING_DIM_RANGE
+from ...nn.emb import EmbeddingSpecification
+from ...nn.modules import TripleREInteraction
+from ...typing import Hint, Initializer, Normalizer
+
+__all__ = [
+    "TripleRE",
+]
+
+
+class TripleRE(ERModel):
+    r"""An implementation of TripleRE related to [yu2021]_.
+
+    This implementation does not use NodePieceRepresentations so far.
+
+    ---
+    citation:
+        author: Yu
+        year: 2021
+        link: https://vixra.org/abs/2112.0095
```

Comment:
double check please

Reply:
This is the correct link. The name seems to be resembling arXiv on purpose, cf. 
> The visual design of viXra.org (but not its content) is a parody of arXiv.org to highlight Cornell University's unacceptable censorship policy. 
> -- source: https://vixra.org/why

Reply:
Oh...